instrumentation/httpd/src/otel/mod_otel.cpp

```diff
@@ -99,6 +104,28 @@ static void opentel_child_created(apr_pool_t*, server_rec*)
   initTracer();
 }
 
+// populates environment variables which can be used by other parts of httpd
+void addEnvVars(apr_table_t *envTable, opentelemetry::trace::SpanContext ctx)
+{
+  union {
+    char bfr[33] = {0};
```

Comment:
Is union member initialization defined behaviour?

Reply:
Yes. As it's very similar to `struct`.